### PR TITLE
Mrc 3123 - Populate model Run Id and calibrate options from project summary

### PR DESCRIPTION
**Pull request by LekanAnni**
## Description

This PR adds populate model run, calibrate options to rehydrate state. 

## Type of version change
_Delete as appropriate_

Major - Minor - Patches - None

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

**Review by EmmaLRussell — CHANGES_REQUESTED** [review]
This works for me, but I think it can be simplified by allowing the post window.reload process to fetch results for you.

File: src/app/static/src/app/utils.ts [hunk 1]
```diff
@@ -367,14 +369,45 @@ const transformPathToHash = (dataset: any) => {
...
+export const constructRehydrateProjectState = async (context: ActionContext<LoadState, RootState>, data: ProjectRehydrateResultResponse) => {
     const files = transformPathToHash({...data.state.datasets});
 
+    context.rootState.modelRun.modelRunId = data.state.model_fit.id
+    context.rootState.modelCalibrate.calibrateId = data.state.calibrate.id
+
+    await Promise.all([
+        context.dispatch("modelRun/getResult", {}, {root: true}),
+        context.dispatch("modelCalibrate/getResult", {}, {root: true})
+    ])
```
[the remark this answers is not in the document]
Comment (LekanAnni) [reply src/app/static/src/app/utils.ts:381]: > 

Thanks, much better 👍

File: src/app/static/src/app/hintVersion.ts [hunk 2]
```diff
@@ -1 +1,2 @@
+
```
Comment (EmmaLRussell) [inline src/app/static/src/app/hintVersion.ts:1]: rogue blank line!

**Review by EmmaLRussell — APPROVED** [review]
Thanks, looks good!

**Review by Nicolas-Dolan — APPROVED** [review]

**Pull request merged**